gitea.yaml

```
@@ -0,0 +1,79 @@
+package:
+  name: gitea
+  version: 1.22.1
+  epoch: 0
+  description: self-hosted git service
+  copyright:
+    - license: MIT
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+      - nodejs
+      - npm
+
+pipeline:
+  - uses: git-checkout
+    with:
+      expected-commit: 35c5192b2572041f14a4d33cbd0128da887b360e
+      repository: https://github.com/go-gitea/gitea
+      tag: v${{package.version}}
+
+  - runs: |
+      set -x
+      npm install --no-save
+      BROWSERSLIST_IGNORE_OLD_DATA=true npx webpack
+
+  - runs: |
+      set -x
+      CC= GOOS= GOARCH= CGO_ENABLED=0 go generate -tags 'bindata timetzdata sqlite sqlite_unlock_notify' ./...
+
+  - uses: go/build
+    with:
+      packages: .
+      ldflags: -X "main.Version=v${{package.version}}"
+      output: gitea
+      tags: netgo,osusergo,bindata,timetzdata,sqlite,sqlite_unlock_notify
+
+subpackages:
+  - name: ${{package.name}}-compat
+    description: "Compatibility package for working with upstream docs"
+    pipeline:
+      - runs: |
+          set -x
+          mkdir -p "${{targets.subpkgdir}}"/usr/local/bin
+          cp docker/rootless/usr/local/bin/docker-entrypoint.sh "${{targets.subpkgdir}}"/usr/local/bin
+          cp docker/rootless/usr/local/bin/docker-setup.sh "${{targets.subpkgdir}}"/usr/local/bin
+          cp docker/rootless/usr/local/bin/gitea "${{targets.subpkgdir}}"/usr/local/bin
+          chmod 755 \
+            "${{targets.subpkgdir}}"/usr/local/bin/docker-entrypoint.sh \
+            "${{targets.subpkgdir}}"/usr/local/bin/docker-setup.sh \
+            "${{targets.subpkgdir}}"/usr/local/bin/gitea
+          mkdir -p "${{targets.subpkgdir}}"/etc/profile.d
+          cp contrib/autocompletion/bash_autocomplete "${{targets.subpkgdir}}"/etc/profile.d/gitea_bash_autocomplete.sh
+          chmod 644 "${{targets.subpkgdir}}"/etc/profile.d/gitea_bash_autocomplete.sh
+          mkdir -p "${{targets.subpkgdir}}"/etc/templates
+          cp docker/rootless/etc/templates/app.ini "${{targets.subpkgdir}}"/etc/templates
+          mkdir -p "${{targets.subpkgdir}}"/app/gitea
+          ln -sf /usr/bin/gitea "${{targets.subpkgdir}}"/app/gitea/gitea
```

```suggestion
          set -x
          install -m 755 -d \
            "${{targets.subpkgdir}}"/usr/local/bin \
            "${{targets.subpkgdir}}"/etc/profile.d \
            "${{targets.subpkgdir}}"/etc/templates \
            "${{targets.subpkgdir}}"/app/gitea

          install -m 755 docker/rootless/usr/local/bin/docker-entrypoint.sh "${{targets.subpkgdir}}"/usr/local/bin
          install -m 755 docker/rootless/usr/local/bin/docker-setup.sh "${{targets.subpkgdir}}"/usr/local/bin
          install -m 755 docker/rootless/usr/local/bin/gitea "${{targets.subpkgdir}}"/usr/local/bin
          install -m 644 contrib/autocompletion/bash_autocomplete "${{targets.subpkgdir}}"/etc/profile.d/gitea_bash_autocomplete.sh
          install -m 644 docker/rootless/etc/templates/app.ini "${{targets.subpkgdir}}"/etc/templates
          ln -sf /usr/bin/gitea "${{targets.subpkgdir}}"/app/gitea/gitea
```

How's this work for you to consolidate these mkdir/chmod/cp steps a bit?